src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040000
+name: Sharpin-related autoinflammatory syndrome
+def: "An auntoinflammatory disease in which the cause of the disease is a mutation in the Sharpin gene." [https://orcid.org/0000-0002-0146-1162, PMID:38609546]
```

Comment:
very small details: 
1) there is a typo in "autoinflammatory"
2) This term follows the [disease_series_by_gene pattern](https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/disease_series_by_gene.yaml), therefore 
   - the definition should follow the definition in this pattern (ie use the word "variation" instead of "mutation"
   - add the xref "MONDO:patterns/disease_series_by_gene" to note that the definition follow this pattern. see MONDO:0010117 as an example

Reply:
Comments addressed.